src/coreclr/jit/compiler.cpp

```diff
@@ -1275,8 +1275,12 @@ void DisplayNowayAssertMap()
             fout = _wfopen(strJitMeasureNowayAssertFile, W("a"));
             if (fout == nullptr)
             {
+#if !defined(HOST_WINDOWS)
+                // TODO: how do we print a `const char16_t*` portably?
```

Comment:
Do these changes need for this to compile in CI/locally? There is an upstream change pending that will resolve them; it would be better to omit them here.

Reply:
Yes, locally in docker, so presumably in CI docker as well.  Without these changes we cannot compile on the image:

```
  /runtime/src/coreclr/jit/compiler.cpp:1278:83: error: length modifier 'w' results in undefined behavior or no effect with 's' conversion specifier [-Werror,-Wformat]
   1278 |                 fprintf(jitstdout(), "Failed to open JitMeasureNowayAssertFile \"%ws\"\n",
        |                                                                                  ~^~
  /runtime/src/coreclr/jit/compiler.cpp:1279:25: error: format specifies type 'wchar_t *' but the argument has type 'LPCWSTR' (aka 'const char16_t *') [-Werror,-Wformat]
   1278 |                 fprintf(jitstdout(), "Failed to open JitMeasureNowayAssertFile \"%ws\"\n",
        |                                                                                  ~~~
        |                                                                                  %s
   1279 |                         strJitMeasureNowayAssertFile);
        |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
  2 errors generated.
  make[3]: *** [jit/CMakeFiles/clrjit_universal_wasm32_x64.dir/build.make:189: jit/CMakeFiles/clrjit_universal_wasm32_x64.dir/compiler.cpp.o] Error 1
```